index.html

```diff
@@ -1919,28 +2062,133 @@ <h5 id="Elemental Traits">Elemental Traits</h5>
                </td>
                <td>Elemental Level 1
                </td>
             </tr>
+            <tr>
+               <td>Shape Element
+               </td>
+               <td>
+                  You may control a small amount of an element (water, fire, air, electricty) 
+                  to move and take a form of your will within 25ft. Think fire dancing in your palm, 
+                  smoke rings, or puddles avoiding your foot.
+               </td>
+               <td>1 AP - No Use Limit
+               </td>
+               <td>Elemental Level 3
+               </td>
+            </tr>
             <tr>
                <td>Control Weather
                </td>
-               <td>You can summon localised weather, making it rain or sunny on command. The
-                  weather reverts to normal after 10 min
+               <td>
+                  You may summon localised weather, such as making it rain or sunny on command. The
+                  weather reverts to normal after 10 minutes. This weather cannot be extreme or harmful.
                </td>
                <td>3 AP - no use limit
                </td>
                <td>Elemental Level 5
                </td>
             </tr>
-         </table>
+            </table>
+            <h5 id="Engineer Traits">Engineer Traits</h5>
+            <table class="center">
+               <tr>
+                  <td><strong>Name</strong>
+                  </td>
+                  <td><strong>Desc</strong>
+                  </td>
+                  <td><strong>Notes</strong>
+                  </td>
+                  <td><strong>Req</strong>
+                  </td>
+               </tr>
+               <tr>
+                  <td>Better sights
+                  </td>
+                  <td>
+                     You can use your INT instead of PER when using ranged weapons such as a bow 
+                     or a crossbow.
+                  </td>
+                  <td>
+                  </td>
+                  <td>Engineer Level 1
+                  </td>
+               </tr>
+               <tr>
+                  <td>Repair
+                  </td>
+                  <td>
+                     You may repair a damaged non magical object. You can repair approximately 
+                     1 cubic foot of material per minute of using this ability, though this will be longer 
+                     if the item/material is more complex.
+                  </td>
+                  <td>3 AP - No Use Limit
+                  </td>
+                  <td>Engineer Level 1
+                  </td>
+               </tr>
+               <tr>
+                  <td>Overdrive
+                  </td>
+                  <td>You may improve the efficiancy of a mechanical object for the next 15min.
+                     This doesn't work on the engineer’s own constructs, as they are already
+                     at the best of their ability
+                  </td>
```

Comment:
I have to ask what this actually is, and what does it do? As a GM I don't know how a player would use this.

Reply:
Was thinking something like making a mechanical door close faster, or making a mill grind faster.  Small RP stuff

Reply:
Ah ok, in that case as it's not a magical class it may make sense to have this be a permenant improvement? Like basically you see a door or a mill and you can spend some time optimising it. Or you find a lock and you can tweak it to be more pick resistant. Call it mechanical modification and it allows you to modify or optimise mechanical systems such as what I just said above?